packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1843,6 +1845,7 @@ export const MAIN_RPC_CONTEXT = {
     QUICK_OPEN_EXT: createProxyIdentifier<QuickOpenExt>('QuickOpenExt'),
     WINDOW_STATE_EXT: createProxyIdentifier<WindowStateExt>('WindowStateExt'),
     NOTIFICATION_EXT: createProxyIdentifier<NotificationExt>('NotificationExt'),
+    NOTIFICATION_MAIN: createProxyIdentifier<NotificationMain>('NotificationMain'),
```

Comment:
Sorry for making life difficult, but at a second glance, it looks like neither of these additions is necessary, either: `NOTIFICATION_MAIN` is already present in the object that contains all of the `MAIN` identifiers, and `NOTIFICATION_EXT` is already present in the object with all of the `EXT` identifiers. I believe that wherever you need to refer to those identifiers, you should do so by referring to object in which they already exist.

Reply:
Not at all, I appreciate the thorough review!